languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -149653,6 +149811,94 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         </rule>
     </category>
     <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+        <rule id="PRENDRE_ACTION" name="prendre action-agir" tags="picky" default="temp_off">
+            <pattern>
+                <token inflected="yes">prendre</token>
+                <token>action</token>
+            </pattern>
+            <message>Une autre expression semble plus appropriée.</message>
+            <suggestion><match no="1" postag="(V.*)" postag_regexp="yes" postag_replace="$1">agir</match></suggestion>
+            <example correction="agissons">Et surtout <marker>prenons action</marker>.</example>
+	    </rule>
+	    <rulegroup id="ADMIN_PRESIDENT" name="administration-gouvernement president" default="temp_off">
+		    <rule>
+			    <antipattern>
+				    <token>administration</token>
+				    <token regexp="yes">de|du|d'|des</token>
+			    </antipattern>
+			    <antipattern>
+				    <token>administration</token>
+				    <token postag="M nonfin" />
+			    </antipattern>
+	            <pattern>
+	                <marker>
+	                    <token skip="-1">administration</token>
+	                </marker>
+	                <token regexp="yes">Washington|Adams|Jefferson|Madison|Monroe|Jackson|Van Buren|Harrison|Tyler|Polk|Eisenhower|Kennedy|Johnson|Nixon|Ford|Carter|Reagan|Bush|Clinton|Obama|Trump|Biden</token>
```

Comment:
The indentation here is a bit suprising, Buren|Harrison|Tyler|Polk|Eisenhower|Kennedy|Johnson|Nixon|Ford|Carter|Reagan|Bush|Clinton|Obama|Trump|Biden</token> showing aligned with the character 0 position.

Reply:
Sorry, the space in the name "Van Buren" is to blame for this. I'll remove "van" and the rule will work as intended and look as expected. ;-)

Reply:
Ahaha it's the President's fault! :p